files/en-us/mozilla/firefox/releases/87/index.html

```diff
@@ -92,7 +92,15 @@ <h4 id="DOM">DOM</h4>
 
 <h3 id="webdriver_conformance_marionette">WebDriver conformance (Marionette)</h3>
 
-<p><em>No changes.</em></p>
+<ul>
+  <li>With the Fission (site-isolation) support finished, all the old code has been removed. As such the preference <code>marionette.actors.enabled</code> is no longer available ({{bug(1669172)}}).</li>
+  <li>WebDriver commands following a call to <code>WebDriver:SwitchToFrame</code> will no longer fail with a "no such window" error if the frame's content hasn't yet finished loading ({{bug(1691348)}}).</li>
+  <li>After a <a
+      href="https://firefox-source-docs.mozilla.org/dom/navigation/nav_replace.html#cross-group-navigations">cross-group
+      page navigation</a>, accessing a previously-retrieved element will now always raise a "stale element" error, and
+    no longer might lead to a "no such element" error ({{bug(1690308)}}).</li>
+  <li><code>Addon:Uninstall</code> now raises an <code>unknown error</code> when the id of the add-on to uninstall is unknown ({{bug(1690308)}}).</li>
```

Comment:
The bug was https://bugzilla.mozilla.org/show_bug.cgi?id=1693022
```suggestion
  <li><code>Addon:Uninstall</code> now raises an <code>unknown error</code> when the id of the add-on to uninstall is unknown ({{bug(1693022)}}).</li>
```

Reply:
Ups. Bad copy and paste. Thanks.